python/mxnet/optimizer.py

```diff
@@ -529,6 +534,69 @@ def update_multi_precision(self, index, weight, grad, state):
         self._update_impl(index, weight, grad, state,
                           multi_precision=use_multi_precision)
 
+
+
+@register
+class Signum(Optimizer):
+    """The SGD optimizer with momentum and weight decay.
```

Comment:
The one line summary should also mention it only takes the sign. Otherwise the readers don't know it until they see line 547

Reply:
added details to the doc accordingly.